packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -254,104 +254,119 @@ class DatadogConfiguration {
-//   /// existing SDK.
-//   ///
-//   /// Defaults to `20.0`.
-//   double tracingSamplingRate;
-
-//   /// Configurations for additional plugins that will be created after Datadog
-//   /// is initialized.
-//   final List<DatadogPluginConfiguration> additionalPlugins = [];
-
-//   DdSdkExistingConfiguration({
-//     this.loggingConfiguration,
-//     this.detectLongTasks = true,
-//     this.longTaskThreshold = 0.1,
-//     this.tracingSamplingRate = 20.0,
-//     List<String>? firstPartyHosts,
-//     this.firstPartyHostsWithTracingHeaders = const {},
-//   }) {
-//     // Attempt a union if both configuration options are present
-//     if (firstPartyHosts != null) {
-//       // make map mutable
-//       firstPartyHostsWithTracingHeaders =
-//           Map<String, Set<TracingHeaderType>>.from(
-//               firstPartyHostsWithTracingHeaders);
-
-//       for (var entry in firstPartyHosts) {
-//         final headerTypes = firstPartyHostsWithTracingHeaders[entry];
-//         if (headerTypes == null) {
-//           firstPartyHostsWithTracingHeaders[entry] = {
-//             TracingHeaderType.datadog
-//           };
-//         } else {
-//           headerTypes.add(TracingHeaderType.datadog);
-//         }
-//       }
-//     }
-//   }
-
-//   void addPlugin(DatadogPluginConfiguration pluginConfiguration) =>
-//       additionalPlugins.add(pluginConfiguration);
-// }
+class DatadogAttachConfiguration {
+  /// Enable or disable detection of "long tasks"
+  ///
+  /// Long task detection attempts to detect when an application is doing too
+  /// much work on the main isolate which could prevent your app from rendering
+  /// at a smooth framerate.
+  ///
+  /// This option does not have any affect on options already set in the Native
+  /// SDK, and only initializes long task detection on the main Dart isolate.
+  ///
+  /// This option is ignored if RUM is not enabled in the Native SDK.
+  ///
+  /// Defaults to true.
+  bool detectLongTasks;
+
+  // The amount of elapsed time that is considered to be a "long task", in /
+  //seconds.
+  ///
+  /// If the main isolate takes more than [longTaskThreshold] seconds to process
+  /// a microtask, it will appear as a Long Task in Datadog RUM Explorer. This
+  /// has a minimum of 0.02 seconds.
+  ///
+  /// This threshold only applies to the Dart long task detector. The Native
+  /// SDKs / will retain their own thresholds.
+  ///
+  /// This option is ignored if RUM is not enabled in the Native SDK.
+  ///
+  /// Defaults to 0.1 seconds
+  final double longTaskThreshold;
+
+  /// Whether to report Flutter specific performance metrics (build and raster
+  /// times)
+  ///
+  /// This uses the [SchedulerBinding.addTimingsCallback] method to report build
+  /// and raster times for views, and has a documented negligible impact on
+  /// performance.
+  ///
+  /// Defaults to false
+  bool reportFlutterPerformance = false;
```

Comment:
should it be a part of this PR? it seems we didn't have it before?

Reply:
We didn't, but that was an oversight. I think adding it here makes sense.